### PR TITLE
Testimonial-card

### PR DESCRIPTION
**Pull request by liepsneletech**

### CODE REVIEW

**Review by Dorenus — DISMISSED** [review]

**Review by Rasosm — DISMISSED** [review]

**Review by EvgenySivaykin — DISMISSED** [review]

**Review by zilva149 — CHANGES_REQUESTED** [review]
Where is blog card code gone? :D

File: html-templates.html [hunk 1]
```diff
@@ -1,115 +1,157 @@
...
+            <div class="right-column">
+                <h5 class="title">First Growing Process</h5>
+                <p class="text">
+                    At vero eos et accusamus etiusto odio praesentium.
+                </p>
+            </div>
+        </div>
+        <!-- FEATURE-BOX-end  -->
+
+        <!-- FORMS-start  -->
```
Comment (zilva149) [inline html-templates.html:80]: Somewhere in between should be blog card

**Review by zilva149 — CHANGES_REQUESTED** [review]

File: html-templates.html (hunk 1 above)
Comment (zilva149) [reply html-templates.html:80]: card is fine by the way

Comment (zilva149) [reply html-templates.html:80]: I think maybe you forgot to do `git merge origin` after my blog-card pull request came through. There are conflicts now as seen at the bottom of this page.

**Pull request closed without merging**